datalad_metalad/add.py

```diff
@@ -196,6 +198,8 @@ class Add(Interface):
     required_additional_keys_lines = "\n".join(
         map(repr, required_additional_keys))
 
+    result_renderer = "generic"
```

Comment:
``meta-add`` does not try to render results, it just yields them and relies on the calling datalad code to do the result rendering. So this seems unnecessary to me, but I might not understand the result rendering infrastructure sufficiently.

This does not include rendering in batch mode, which is an independent protocol that does is not bound to the result dictionary conventions, and actually does not adhere to them. For example, it does not contain a ``path``-key.